code/__DEFINES/combat.dm

```diff
@@ -6,7 +6,7 @@
 /// Physical fracturing and warping of the material.
 #define BRUTE		"brute"
 /// Scorching and charring of the material.
-#define BURN		"fire"
+#define FIRE		"fire"
```

Comment:
Likely the remains of two seperate damage types but the defines were just made the same for less file changes.